test/integration/ClientProxyTest.js

```diff
@@ -43,26 +46,63 @@ describe('ClientProxyTest', function () {
     });
 
     it('client without active connection should return unknown version', function () {
-        const connectionManagerStub = sandbox.stub(ClientConnectionManager.prototype);
-        connectionManagerStub.getActiveConnections.returns({});
```

Comment:
Just curious, for getActiveConnections() why we say returns {} ? I think it should return an array. I could not understand it

Reply:
That's probably a misprint. Should be `[]`